### DISCUSSION TITLE
Enhance/New Source Connector for Plaid Investments

### DISCUSSION THREAD

**Discussion by sentinel-ai** [airbytehq/airbyte#36098] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
Requesting a source connector for Plaid Investments (the current Plaid connector only supports Balances/Holdings)
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
No, latest version is fine.

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
We are trying to use Airbyte to simplify our data ingestion for Plaid Investments.  We use the data for adding insights on investments/trades that help investment returns.  

* How often do you want to run syncs?
We intend to run the sync initially every 24 hours and perhaps more frequently later on (upto every hour) 

* If this is an API source connector, which entities/endpoints do you need supported?
Plaid Investments (containing both InvestmentHoldingStream and IncrementalInvestmentTransactionStream
* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 
You are free to name us as a mutual user (Sentinel AI, Inc.)

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
We use Plaid API directly which is a uses fewer pieces of common code and is harder/more complex to support.

## Are you willing to submit a PR?
Potentially (with limited resourcing, this has fallen behind).

**Comment by sentinel-ai** [comment]
We actually have this working but would appreciate help in submitting PR/ensuring we are running tests properly (IT@sentinelrisk.ai)

**Comment by sajarin** [comment]
Hey @sentinel-ai, our team would be happy to review the PRs. How can we help?

[a comment by an automated account is not shown]
